pkg/apis/cincinnati/v1beta1/cincinnati_types.go

```diff
@@ -25,6 +25,13 @@ type CincinnatiSpec struct {
 	// GraphDataImage is a container image that contains the Cincinnati graph
 	// data. The data is copied to /var/lib/cincinnati/graph-data.
 	GraphDataImage string `json:"graphDataImage"`
+
+	// IPv6 defines whether the services bind to "0.0.0.0" or to "::"
+	IPv6 bool `json:"ipv6,omitempty"`
```

Comment:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#primitive-types

> Think twice about bool fields. Many ideas start as boolean but eventually trend towards a small set of mutually exclusive options. Plan for future expansions by describing the policy options explicitly as a string type alias (e.g. TerminationMessagePolicy).

My gut feeling is that adding the `Address` to the spec is sufficient. That way we can default to '0.0.0.0' or force it to be defined.

Reply:
The first implementation was done in that way, then I changed it based on Ryan's suggestions, anyway I'd say a boolean makes sense, a user won't likely know which address is going to get the pod when created, and between defining a property with the bind-address or just setting a boolean to decide if the services listen on IPv6 or IPv4 I'd rather go with the second option, but again, I'm open to suggestions.

Reply:
I don't understand.

> a user won't likely know which address is going to get the pod when created

Then why would we bother taking the `Address` on the spec?

Reply:
I mean, if you add the Address to the spec so users can choose which address the services bind to, they will likely end using "0.0.0.0" or "::", the IP address that the SDN will assign to the pod is not something you can know beforehand easily. That's why I thought a boolean makes more sense.

Reply:
Binding to "::" works on IPv4 and IPv6 clusters. I reverted the changes from the API types.